static/js/directives/screenshare.js

```diff
@@ -180,8 +180,21 @@ define(['jquery', 'underscore', 'text!partials/screenshare.html', 'text!partials
 						$scope.stopScreenshare();
 					}
 				}, function(err) {
-					console.log("Screen sharing request returned error", err);
-					alertify.dialog.alert(translation._("Failed to start screen sharing (%s).", err));
+					var errMsg = '';
```

Comment:
do not define as empty string and provide reference where this error handling comes from? When does err.detail ever exist?